nservicebus/azure-service-bus/keep-connectionstring-safe.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Keep connection string safe
+summary: Configuring Azure ServiceBus transport to secure connection strings. 
+component: ASB
+tags:
+- Cloud
+- Azure
+- Transports 
+- Security
+reviewed: 2016-04-26
+---
+
+Transport, versions 6 and below, uses raw connection string provided into message headers.  
+For instance, a typical value for `ReplyTo` header could be:  
+
+`[queue name]@Endpoint=sb://[namespace name].servicebus.windows.net/;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=[secrets here]`
+  
+It could result in a leak of sensitive information, f.i. logging it.  
```

Comment:
how about 

> This could potentially result in a leak of sensitive information, for example in [log files](/nservicebus/logging/) or in the [error queue](/nservicebus/errors/).